### PR TITLE
Update with release info for 1.11.0.

### PR DESCRIPTION
**Pull request by allisonvacanti**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,76 @@
...
+
+- NVIDIA/cub#201: The intermediate accumulator type used when `DeviceScan` is
+  invoked with different input/output types is now consistent
+  with [P0571](https://wg21.link/P0571). This may produce different results for
+  some edge cases when compared with earlier releases of CUB.
+
+## New Features
+
+- NVIDIA/cub#204: Faster `DeviceRadixSort`, up to 2x performance increase for
+  32/64-bit keys on SM70+. Thanks to Andy Andinets for this contribution.
```
Comment (canonizer) [inline CHANGELOG.md:27]: The new implementation works on SM60+.

Comment (canonizer) [inline CHANGELOG.md:27]: Misprint: "Andy Adinets" is the correct spelling.

**Review by canonizer — DISMISSED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (allisonvacanti) [reply CHANGELOG.md]: Updated.

Comment (allisonvacanti) [reply CHANGELOG.md]: Updated.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,76 @@
...
+
+- NVIDIA/cub#201: The intermediate accumulator type used when `DeviceScan` is
+  invoked with different input/output types is now consistent
+  with [P0571](https://wg21.link/P0571). This may produce different results for
+  some edge cases when compared with earlier releases of CUB.
+
+## New Features
+
+- NVIDIA/cub#204: Faster `DeviceRadixSort`, up to 2x performance increase for
+  32/64-bit keys on SM60+. Thanks to Andy Adinets for this contribution.
```
Comment (brycelelbach) [inline CHANGELOG.md:27]: Can we say "SM60+ (Pascal and up)"? Some of our users will be familiar with our architectures by name only.

**Review by brycelelbach — APPROVED** [review]
This looks great, excellent work!

File: CHANGELOG.md (hunk 2 above)
Comment (allisonvacanti) [reply CHANGELOG.md:27]: Good point, updated.

**Pull request merged**